recipes/scikit-fda/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "scikit-fda" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-fda-{{ version }}.tar.gz
+  sha256: 69f683b866813d61e209f2213ebbe266b3762588569f39a9b0065b950ba260f2
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - cython
+    - dcor
+    - fdasrsf >=2.2.0
+    - findiff
+    - lazy_loader
+    - matplotlib-base
+    - multimethod >=1.5
+    - numpy >=1.16
+    - pandas >=1.0
+    - python
+    - rdata
+    - scikit-datasets >=0.1.24
+    - scikit-learn >=0.20
+    - scipy >=1.3.0
+    - typing-extensions
+
+test:
+  imports:
+    - docs
+    - skfda
+    - tutorial
```

Comment:
Oof, it would be really good to fix up the packaging so that `docs` and `tutorial` aren't installed as packages, which potentially conflict with those names from other packages.